.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    if: github.repository == 'pytest-dev/pytest-timeout'
```

Comment:
why this if statement?  this workflow is in this repo, or are there other ways to trigger this from outside this repo?

Reply:
probably same as https://github.com/python-attrs/attrs/pull/1179

Reply:
Thanks for the link (is there an open source repo which @hugovk **does not** contribute to? Still to find one hehehe!), however here the situation is a bit different, as this workflow only triggers on `release`, meaning it would only trigger if a fork created a release on the fork, which seems strange/rare.

But seems harmless to leave.